client/src/TodoList/TodoListContainer.js

```diff
@@ -9,7 +9,7 @@ class App extends Component {
   };
 
   async componentDidMount() {
-    this.getCurrentUser();
+    // this.getCurrentUser();
```

Comment:
will move this validation check into the routes component did mount in order to check if a user is logged in or not. Maybe not the component did mount but somewhere around there